deno_dist/helper/dev/index.ts

```diff
@@ -17,7 +17,7 @@ const handlerName = (handler: Function) => {
   return handler.name || (isMiddleware(handler) ? '[middleware]' : '[handler]')
 }
 
-export const inspectRoutes = (hono: Hono): RouteData[] => {
+export const inspectRoutes = <TEnv extends Env>(hono: Hono<TEnv>): RouteData[] => {
```

Comment:
We use `E` instead of `TEnv` in the same cases. So it would be better to use `E`.

https://github.com/honojs/hono/blob/5e0ef3048b5add5dadbbb4e936254b71058351b2/src/hono-base.ts#L53

Reply:
Ah, my mistake. Replaced all occurrences of `TEnv` with `E` & re-run the `denoify` script.